### ISSUE TITLE
[SF][ui5-badge] add a new colorScheme

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#3389]
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.

add a new colorScheme for badge component

![Screen Shot 2021-06-04 at 9 48 20 AM](https://user-images.githubusercontent.com/78520316/120734260-27f3a580-c51b-11eb-93c4-f179dd84d64b.png)

**Comment by dobrinyonkov** [comment]
Hello @sfsf-xweb-sh, 

is your feature request to add a new colorScheme option with different border color (`--sapAccentColor10: #647987`) but the same `background-color` like `color-scheme=10` ?

Kind Regards,
Dobrin

**Comment by sfsf-xweb-sh** [comment]
Hi @dobrinyonkov 
Yes

**Comment by sfsf-xweb-sh** [comment]
Hi @dobrinyonkov 
Please add another color Scheme
<img width="657" alt="Screen Shot 2021-06-09 at 6 11 03 PM" src="https://user-images.githubusercontent.com/78520316/121337018-a783d900-c94e-11eb-95fa-646705b74527.png">

**Comment by dobrinyonkov** [comment]
Hello, this is a feature request to add new color schemes to the `ui5-badge`. 
I guess it would be better to add the possibility for customization.

Kind Regards,
Dobrin

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh 
could you provide the design for the color schemes that are missing currently.

Also, you can also change both the background and the border with the css vars you would like:

```html
<ui5-badge>text</ui5-badge>
```

```css
ui-5badge {
backgorund-color:
border-color:
}
```

But if there are color schemes that are present in Fiori 3 and OpenUI5 InfoLabel (the equivalent of ui5-badge) we will definitely add them.

**Comment by sfsf-xweb-sh** [comment]
Hi @ilhan007 

The background-color: #f8de7
color is: #7ca10c

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh 

Currently, the Badge has 10 built-in color schemas and to add new color schema we need to have it specified in Fiori 3.
I could not find your schema in the standard ones. You can have a look at the OpenUI5 equivalent as well:
https://openui5nightly.hana.ondemand.com/entity/sap.tnt.InfoLabel/sample/sap.tnt.sample.InfoLabel

If your color schema is a standard existing one, please provide a link to Fiori 3 design papers or UI5 official samples.
If not, you need to file a request to the Visual Design Team.


Also, consider applying the color schema yourself. The Badge can be easily customized with standard CSS. Just use the CSS Variables you need

```css
ui5-badge {
   backgorund-color:
    border-color:
}
```

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd

could you check if the built-in color schemas are all by the Fiori 3 spec and if we miss one?

BR,
ilhan

**Comment by sfsf-xweb-sh** [comment]
Hi @ilhan007 
Is there a timeline of this component ?

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd

could you assist with the this issue by checking if the built-in color schemas, currently available in the Badge web component (InfoLabel control in OpenUI5) are all correct by the Fiori 3 spec, using the correct CSS vars, and if we miss one?

BR,
ilhan

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh did you read my previous comments, in case not I will paste it here:

Currently, the Badge has 10 built-in color schemas and to add new color schema we need to have it specified in Fiori 3.
I could not find your schema in the standard ones. You can have a look at the OpenUI5 equivalent as well:
https://openui5nightly.hana.ondemand.com/entity/sap.tnt.InfoLabel/sample/sap.tnt.sample.InfoLabel

If your color schema is a standard existing one, please provide a link to Fiori 3 design papers or UI5 official samples.
If not, you need to file a request to the Visual Design Team. If this is the case, just confirm and we can contact offline to continue with the process of involving the Visual Design Team.

Also, consider applying the color schema yourself. The Badge can be easily customized with standard CSS. Just use the CSS Variables you need

ui5-badge {
   backgorund-color:
    border-color:
}

**Comment by sfsf-xweb-sh** [comment]
Hi @ilhan007 

I will communicate with our ux to see whether this feature request is a standard  one

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd it seems the newest design changes (for couple of color schemes) are not adopted. @alexandar-mitsev already checked and confirmed that. Could you adopt the design.
I think this the design page for the InfoLabel: https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2036835836 but you can double check with the VD.

**Comment by alexandar-mitsev** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

There is also a mistake in the border colors of accent 9 and accent 10 which can be solved with the adoption of the new design.

Regards

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh  the badge design has been updated to the latest spec available with 0.31.14
https://wiki.wdf.sap.corp/wiki/display/CPDesign/Info+Label



<img width="280" alt="Screenshot 2021-08-03 at 12 13 39" src="https://user-images.githubusercontent.com/15702139/127990473-bb7cc36c-c9ce-4c2e-ae90-1a07f12b07cf.png">


**Note:** There are still discussions within the Visual Design team, if this is the final spec, in case something changes we will update the design accordingly:
